spec/javascripts/components/story/expanded_story/expanded_story_state_spec.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExpandedStoryState from 'components/story/ExpandedStory/ExpandedStoryState';
+import { states } from '../../../../../app/assets/javascripts/models/beta/story';
+
+describe('<ExpandedStoryState />', () => {
+  it('renders component title', () => {
+    const story = { state: 'started', _editing: { state: 'started' } };
+
+    const wrapper = shallow(<ExpandedStoryState story={story} />);
+
+    expect(wrapper.text()).toContain(I18n.t('activerecord.attributes.story.state'));
+  });
+
+  describe("states at select", () => {
+    states.map(state => {
+      it(`sets select value as ${state}`, () => {
+        const story = {
+          _editing: { state: state }
+        };
+
+        const wrapper = shallow(<ExpandedStoryState story={story} />);
+        const select = wrapper.find('select');
+
+        expect(select.prop('value')).toBe(state);
+      });
+    });
+  });
+
+  describe('onChange', () => {
```

Comment:
It's kinda :nail_care:, but what do you think about creating a `context` here to better describe the component's behavior? Ex:

```js
context('when the user selects a state', () => {
  it('triggers the edit callback passing the story description', () => { ... })
})
```